### DISCUSSION TITLE
Getting a new error since last update - “ serrver compliant with your settings not found, ”

### DISCUSSION THREAD

**Discussion by Gitohero** [haugene/docker-transmission-openvpn#2864] [General]
Hi - all just starting to get this error in the last day or so, it might coincide with the latest update. The error log is below,

2024/07/20 23:14:04 | stdout | [41mERROR:[0m server compliant with your settings not found, review them
2024/07/20 23:14:04 | stdout | Best server :
2024/07/20 23:14:04 | stdout | WARNING: OVPN: unable to find a server with the specified parameters, please review your parameters, NORDVPN_COUNTRY=Canada, NORDVPN_CATEGORY=legacy_p2p, NORDVPN_PROTOCOL=tcp
2024/07/20 23:14:04 | stdout | parse error: Invalid numeric literal at line 1, column 20
2024/07/20 23:14:04 | stdout | INFO: OVPN:Searching for technology: openvpn_tcp
2024/07/20 23:14:04 | stdout | INFO: OVPN: Searching for group: legacy_p2p
2024/07/20 23:14:04 | stdout | INFO: OVPN: Searching for country : Canada (38)
2024/07/20 23:14:04 | stdout | INFO: OVPN:Selecting the best server...
2024/07/20 23:14:04 | stdout | Checking NORDPVN API responses
2024/07/20 23:14:03 | stdout | INFO: OVPN: Removing existing configs in /etc/openvpn/nordvpn
2024/07/20 23:14:03 | stdout | INFO: OVPN: ok, configurations download site reachable
2024/07/20 23:14:02 | stdout | INFO: OVPN: DNS resolution ok
2024/07/20 23:14:02 | stdout | INFO: OVPN: Checking curl installation
2024/07/20 23:14:02 | stdout | /etc/openvpn/nordvpn/..
2024/07/20 23:14:02 | stdout | Executing setup script for NORDVPN
2024/07/20 23:14:02 | stdout | Provider NORDVPN has a bundled setup script. Defaulting to internal config
2024/07/20 23:14:02 | stdout | Running with VPN_CONFIG_SOURCE auto
2024/07/20 23:14:02 | stdout | Using OpenVPN provider: NORDVPN
2024/07/20 23:14:02 | stdout | Creating TUN device /dev/net/tun
2024/07/20 23:14:02 | stdout | TRANSMISSION_HOME is currently set to: /config/transmission-home

What am I doing wrong? It was working fine a week ago. Thank you

**Comment by edgd1er** [comment]
I don't know what happened but I can say what is happening:
* 2024/07/20 23:14:04 | stdout | �[41mERROR:�[0m server compliant with your settings not found, review them
2024/07/20 23:14:04 | stdout | Best server :
2024/07/20 23:14:04 | stdout | WARNING: OVPN: unable to find a server with the specified parameters, please review your parameters, NORDVPN_COUNTRY=Canada, NORDVPN_CATEGORY=legacy_p2p, NORDVPN_PROTOCOL=tcp
=> according to your settings, the api could not suggest a server, or the api did not answer at all to your requests.

* 2024/07/20 23:14:04 | stdout | parse error: Invalid numeric literal at line 1, column 20 => api is not responding anymore, too many requests were performed due to the container being restarted continuously

What to do:
* stop the container, wait for a while (5-10 minutes). Nordvpn has a throttle policy to protect its api performance. test once, if connection is still failing, then wait for another 15 minutes.
* set DEBUG to true, to view all commands

**Reply by Gitohero** [reply]
Thank you kindly - but now getting this error:

2024/07/21 00:58:01 | stdout | /etc/openvpn/start.sh: line 225: TRANSMISSION_RPC_USERNAME: unbound variable
2024/07/21 00:58:01 | stdout | Setting OpenVPN credentials...
2024/07/21 00:58:01 | stdout | Modification: Updating status for config failure detection
2024/07/21 00:58:01 | stdout | Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop
2024/07/21 00:58:01 | stdout | Modification: Set output verbosity to 3
2024/07/21 00:58:01 | stdout | Modification: Change tls-crypt keyfile path
2024/07/21 00:58:01 | stdout | Modification: Update/set resolv-retry to 15 seconds
2024/07/21 00:58:01 | stdout | Modification: Change ping options
2024/07/21 00:58:01 | stdout | Modification: Change ca certificate path
2024/07/21 00:58:01 | stdout | Modification: Point auth-user-pass option to the username/password file
2024/07/21 00:58:01 | stdout | Modifying /etc/openvpn/nordvpn/ca1663.nordvpn.com.ovpn for best behaviour in this container
2024/07/21 00:58:00 | stdout | Starting OpenVPN using config ca1663.nordvpn.com.ovpn
2024/07/21 00:58:00 | stdout | OVPN: NORDVPN: selected: ca1663.nordvpn.com, VPN_PROVIDER_HOME: /etc/openvpn/nordvpn
2024/07/21 00:58:00 | stdout | INFO: OVPN: Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_tcp/servers/ca1663.nordvpn.com.tcp.ovpn
2024/07/21 00:58:00 | stdout | INFO: OVPN: Downloading config: ca1663.nordvpn.com.ovpn
2024/07/21 00:58:00 | stdout | Best server : ca1663.nordvpn.com
2024/07/21 00:58:00 | stdout | INFO: OVPN: Best server : ca1663.nordvpn.com, load: null
2024/07/21 00:57:59 | stdout | INFO: OVPN:Searching for technology: openvpn_tcp
2024/07/21 00:57:59 | stdout | INFO: OVPN: Searching for group: legacy_p2p
2024/07/21 00:57:59 | stdout | INFO: OVPN: Searching for country : Canada (38)
2024/07/21 00:57:59 | stdout | INFO: OVPN:Selecting the best server...
2024/07/21 00:57:59 | stdout | Checking NORDPVN API responses
2024/07/21 00:57:58 | stdout | INFO: OVPN: Removing existing configs in /etc/openvpn/nordvpn
2024/07/21 00:57:58 | stdout | INFO: OVPN: ok, configurations download site reachable
2024/07/21 00:57:56 | stdout | INFO: OVPN: DNS resolution ok
2024/07/21 00:57:56 | stdout | INFO: OVPN: Checking curl installation
2024/07/21 00:57:53 | stdout | /etc/openvpn/nordvpn/..
2024/07/21 00:57:53 | stdout | Executing setup script for NORDVPN
2024/07/21 00:57:53 | stdout | Provider NORDVPN has a bundled setup script. Defaulting to internal config
2024/07/21 00:57:53 | stdout | Running with VPN_CONFIG_SOURCE auto
2024/07/21 00:57:53 | stdout | Using OpenVPN provider: NORDVPN
2024/07/21 00:57:52 | stdout | Creating TUN device /dev/net/tun
2024/07/21 00:57:48 | stdout | TRANSMISSION_HOME is currently set to: /config/transmission-home

**Comment by edgd1er** [comment]
set TRANSMISSION_RPC_USERNAME to '' should fix the problem, maybe also TRANSMISSION_RPC_PASSWORD

**Reply by Gitohero** [reply]
THANK YOU - just to be clear, this means going to settings adding two line items like the ones you entered above, and keeping them blank ‘’? 

<img width="584" alt="image" src="https://github.com/user-attachments/assets/b163ac87-8da7-43bb-8e75-612b86b56f97">

**Reply by edgd1er** [reply]
yes, that's what I would do in command line, in your interface I don't know.
if you have portainer or can enter into the running container, you could check env vars.

**Reply by Gitohero** [reply]
thank you kindly for your support
But still getting this error pls

2024/07/21 08:48:50 | stdout | RTNETLINK answers: Operation not permitted
2024/07/21 08:48:50 | stdout | adding route to local network 192.168.2.0/24 via 172.23.0.1 dev eth0
2024/07/21 08:48:50 | stdout | Setting OpenVPN credentials...
2024/07/21 08:48:50 | stdout | Modification: Updating status for config failure detection
2024/07/21 08:48:50 | stdout | Modification: Remap SIGUSR1 signal to SIGTERM, avoid OpenVPN restart loop
2024/07/21 08:48:50 | stdout | Modification: Set output verbosity to 3
2024/07/21 08:48:50 | stdout | Modification: Change tls-crypt keyfile path
2024/07/21 08:48:50 | stdout | Modification: Update/set resolv-retry to 15 seconds
2024/07/21 08:48:50 | stdout | Modification: Change ping options
2024/07/21 08:48:50 | stdout | Modification: Change ca certificate path
2024/07/21 08:48:50 | stdout | Modification: Point auth-user-pass option to the username/password file
2024/07/21 08:48:50 | stdout | Modifying /etc/openvpn/nordvpn/ca1515.nordvpn.com.ovpn for best behaviour in this container
2024/07/21 08:48:50 | stdout | Starting OpenVPN using config ca1515.nordvpn.com.ovpn
2024/07/21 08:48:50 | stdout | OVPN: NORDVPN: selected: ca1515.nordvpn.com, VPN_PROVIDER_HOME: /etc/openvpn/nordvpn
2024/07/21 08:48:50 | stdout | INFO: OVPN: Downloading from: https://downloads.nordcdn.com/configs/files/ovpn_tcp/servers/ca1515.nordvpn.com.tcp.ovpn
2024/07/21 08:48:50 | stdout | INFO: OVPN: Downloading config: ca1515.nordvpn.com.ovpn
2024/07/21 08:48:50 | stdout | Best server : ca1515.nordvpn.com
2024/07/21 08:48:50 | stdout | INFO: OVPN: Best server : ca1515.nordvpn.com, load: null
2024/07/21 08:48:49 | stdout | INFO: OVPN:Searching for technology: openvpn_tcp
2024/07/21 08:48:49 | stdout | INFO: OVPN: Searching for group: legacy_p2p
2024/07/21 08:48:49 | stdout | INFO: OVPN: Searching for country : Canada (38)
2024/07/21 08:48:49 | stdout | INFO: OVPN:Selecting the best server...
2024/07/21 08:48:49 | stdout | Checking NORDPVN API responses
2024/07/21 08:48:48 | stdout | INFO: OVPN: Removing existing configs in /etc/openvpn/nordvpn
2024/07/21 08:48:48 | stdout | INFO: OVPN: ok, configurations download site reachable
2024/07/21 08:48:47 | stdout | INFO: OVPN: DNS resolution ok
2024/07/21 08:48:47 | stdout | INFO: OVPN: Checking curl installation
2024/07/21 08:48:46 | stdout | /etc/openvpn/nordvpn/..
2024/07/21 08:48:46 | stdout | Executing setup script for NORDVPN
2024/07/21 08:48:46 | stdout | Provider NORDVPN has a bundled setup script. Defaulting to internal config
2024/07/21 08:48:46 | stdout | Running with VPN_CONFIG_SOURCE auto
2024/07/21 08:48:46 | stdout | Using OpenVPN provider: NORDVPN
2024/07/21 08:48:46 | stdout | Creating TUN device /dev/net/tun
2024/07/21 08:48:45 | stdout | TRANSMISSION_HOME is currently set to: /config/transmission-home

**Comment by edgd1er** [comment]
if syno:
  https://github.com/haugene/docker-transmission-openvpn/issues/1795
else:
 [check elevated privileges.](https://haugene.github.io/docker-transmission-openvpn/faq/)

**Comment by Gitohero** [comment]
Thank you sooo much - that worked!
